seqr/management/tests/load_rna_seq_tpm_tests.py

```diff
@@ -23,31 +23,31 @@ def test_command(self, mock_open, mock_logger, mock_utils_logger, mock_gzip_open
         mock_gzip_file = mock_gzip_open.return_value.__enter__.return_value
         mock_gzip_file.__iter__.return_value = [
             '',
-            'NA19675_D2\t\tENSG00000240361\t12.6\t\n',
-            'NA19678_D1\t\tENSG00000233750\t 6.04\twhole_blood\n',
-            'NA19678_D1\t\tENSG00000240361\t0.0\twhole_blood\n',
-            'NA19677\t\tENSG00000240361\t0.0\tinvalid\n',
-            'GTEX-001\t\tENSG00000240361\t3.1\tinvalid\n',
-            'NA19675_D2\t\tENSG00000233750\t1.04\tmuscle\n',
-            'NA19677\t\tENSG00000233750\t5.31\tmuscle\n',
-            'GTEX-001\t\tENSG00000233750\t7.8\tmuscle\n',
+            'NA19675_D2\t1kg project nåme with uniçøde\t\tENSG00000240361\t12.6\t\n',
+            'NA19678_D1\t1kg project nåme with uniçøde\t\tENSG00000233750\t 6.04\twhole_blood\n',
+            'NA19678_D1\t1kg project nåme with uniçøde\t\tENSG00000240361\t0.0\twhole_blood\n',
+            'NA19677\t1kg project nåme with uniçøde\t\tENSG00000240361\t0.0\tinvalid\n',
+            'GTEX-001\t1kg project nåme with uniçøde\t\tENSG00000240361\t3.1\tinvalid\n',
+            'NA19675_D2\t1kg project nåme with uniçøde\t\tENSG00000233750\t1.04\tmuscle\n',
+            'NA19677\t1kg project nåme with uniçøde\t\tENSG00000233750\t5.31\tmuscle\n',
+            'GTEX-001\t1kg project nåme with uniçøde\t\tENSG00000233750\t7.8\tmuscle\n',
```

Comment:
Add a row with a sample_id that is present in our fixture data but specify a different project for it. For example, maybe include another row for  `NA19678_D1` but with `Test Reprocessed Project` as the project column, and then confirm that there is a warning returned to the user about it and that no data was created from that row

Reply:
Added `mock_gzip_file.__iter__.return_value.append('NA19678_D1\tTest Reprocessed Project\t\tENSG00000240361\t0.2\twhole_blood\n')` a few lines below.

Reply:
It should also be possible. But adding it below can reduce possible churns with the existing tests.